docs/content/handover-between-server-workers.md

```diff
@@ -0,0 +1,52 @@
+
+**Warning** <br/>
+This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the GDK Core is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use](https://github.com/spatialos/UnityGDK/blob/master/README.md#recommended-use).
```

Comment:
"GDK Core"

Reply:
Also this is appearing as a big heading for some reason - I'd change it to just be a callout, i.e. add ">" at the start and lose the "Warning" heading:

> This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the GDK Core is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use](https://github.com/spatialos/UnityGDK/blob/master/README.md#recommended-use).